packages/network-of-terms-query/src/index.ts

```diff
@@ -0,0 +1,12 @@
+export * from './query';
+export * from './catalog';
+export * from './lookup/lookup';
+export * from './terms';
+export * from './search/query-mode';
+export * from './distributions';
+export * from './helpers/logger';
+export * from './server-test';
```

Comment:
`server-test` looks like a file that's only needed in development mode (and that depends on 'devDependencies' packages). If we export it like this, will this work when running in production mode?

Reply:
You’re right that calling `server-test` functions in production code will fail.

Still, I ended up with this solution because it keeps things relatively easy. A purer way to solve this would be to create a separate `network-of-terms-tests` package and depend on that as a devDependency. I considered that overkill for now.

Reply:
👍